libs/designer/src/lib/core/configuretemplate/utils/helper.ts

```diff
@@ -219,3 +219,32 @@ export const getSupportedSkus = (connections: Record<string, Template.Connection
 export const getDefinitionFromWorkflowManifest = (manifest: Template.WorkflowManifest): LogicAppsV2.WorkflowDefinition => {
   return (manifest?.artifacts?.find((artifact) => equals(artifact.type, 'workflow')) as any)?.file as LogicAppsV2.WorkflowDefinition;
 };
+
+export const getSaveMenuButtons = (
+  resourceStrings: Record<string, string>,
+  currentStatus: Template.TemplateEnvironment,
+  onSave: (status: Template.TemplateEnvironment) => void
+): { text: string; onClick: () => void }[] => {
+  const isPublishedState = currentStatus === 'Testing' || currentStatus === 'Production';
```

Comment:
It is better to always use equals() helper method to avoid case sensitive issues